### DISCUSSION TITLE
[Bug] :beetle: emoji (1fab2.png) gives a 404 Error on githubassets.com

### DISCUSSION THREAD

**Discussion by justwheel** [community/community#7470] [Other Feature Feedback, Questions, & Ideas]
# Summary

The `:beetle:` 🪲 emoji does not render when used across github.com

# Background

I am a fan of tagging my commits with emojis, so I have noticed that the `:beetle:` emoji has not appeared on github.com for some time. I can see it when writing an issue, presumably because the browser is rendering the emoji locally. But once the content makes its way onto github.com, the `:beetle:` shows up as, well, `:beetle:`. Take a look:

![Screenshot of a file in a git repository where the :beetle: emoji does not render](https://user-images.githubusercontent.com/4721034/140771294-d69e39ee-d5d8-4141-bc0b-cc007db8d086.png "Screenshot of a file in a git repository where the :beetle: emoji does not render")

I have not seen this issue with other emojis. Only the beetle emoji.

# Details

It appears the Unicode PNG is missing from the githubassets.com. If I try to open the image of the beetle in a new tab, I get this link, which in turn gives a 404 Error:

https://github.githubassets.com/images/icons/emoji/unicode/1fab2.png

But I can find other emoji that appear correctly, like this camera emoji:

https://github.githubassets.com/images/icons/emoji/unicode/1f4f8.png

# Outcome

`:beetle:` emoji makes a reappearance on github.com

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
